packages/lune/src/config.ts

```diff
@@ -1,26 +1,45 @@
 import type { Plugin } from "esbuild";
-
+import type { UserConfig } from "@farmfe/core";
 import { existsSync } from "fs";
 import { resolve, parse } from "path";
 import { pathToFileURL } from "url";
 
-export interface LuneCfg {
-  /**
-   * If CSS Modules should be enabled
-   * @default false
-   */
-  cssModules?: boolean | "legacy";
+export type LuneCfg = {
   /**
    * If the output should be minified
    * @default false
    */
   minify?: boolean;
+} & (
+  | {
+      /**
+       * If Farm should be used instead of esbuild
+       */
+      useFarm?: false;
 
-  /** esbuild plugins that run before Lune's transforms */
-  prePlugins?: Plugin[];
-  /** esbuild plugins that run after Lune's transforms */
-  postPlugins?: Plugin[];
-}
+      /**
+       * If CSS Modules should be enabled
+       * @default false
+       */
+      cssModules?: boolean | "legacy";
```

Comment:
we really could do with css modules support for farm, is it built in?

Reply:
yes